core/src/main/scala/ai/lum/odinson/lucene/search/OdinsonIndexSearcher.scala

```diff
@@ -44,4 +49,43 @@ class OdinsonIndexSearcher(
     search(query, manager)
   }
 
+  def pureSearch(query: Query, n: Int): OdinResults = {
+
+    // Modify query to only return sentence entries
+    val builder = new BooleanQuery.Builder()
```

Comment:
add tests for the pureSearch:
- doesnt retrieve type: parent
- continuation works
- positive flow works (simpoe/or query)

Reply:
tests added in `ai.lum.odinson.lucene.search.OdinsonIndexSearcherTest`